invenio/b2share/modules/b2deposit/restful.py

```diff
@@ -216,13 +222,28 @@ def get(self, oauth, domain_name, **kwargs):
         record_list = []
         start = page_offset * page_size
         stop = start + page_size
+
         for record_id in record_ids[start : stop]:
             record_details = get_record_details(record_id, curr_user_email)
             record_list.append(record_details)
-        if stop < len(record_ids):
-            record_list.append('...') # continuation indicator
-        return record_list
 
+        if len(record_list) == 0:
```

Comment:
All this long list of ifs seems unnecessary to me. Isn't it equivalent with:

`if len(record_list) == 0: return jsonify({}); else: return jsonify(Deposits=record_list)` ?

but it might be that I'm missing something, please comment